catalog-info.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sda-spring-boot-commons
+  annotations:
+    sda.se/summary: |
+      The [sda-spring-boot-commons](https://github.com/SDA-SE/sda-spring-boot-commons) library
+      includes several modules to speed up SDA-compliant microservice development. It is
+      based on the [Spring Boot](https://spring.io/projects/spring-boot) framework. It provides
+      several starter modules for additional functionality. Those starter modules configure the
+      framework with default SDA specific settings and add support for an easy and fast integration
+      of standard technologies such as MongoDB, S3 or Kafka.
+    sda.se/federation-scopes: public
+  description: |
+    SDA Spring Boot Commons is a set of libraries to bootstrap services easily that follow
+    the patterns and specifications promoted by the SDA SE.
+  links:
+    - url: https://github.com/SDA-SE/sda-spring-boot-commons
+      title: GitHub
+      icon: code
+spec:
+  type: library
+  lifecycle: production
+  system: sda-spring-boot-commons
+  owner: bedrock
+---
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: sda-spring-boot-commons
```

Comment:
In the long term we (s|c)ould have a common system for backend libraries (as proposed [here](https://github.com/SDA-SE/sda-kustomize-components/pull/2#discussion_r1011382265)).

Should we prioritize a task to define a structure to avoid moving too many components later?

Reply:
It already exists: PLP-471.